src/makeRobotAccountant.js

```diff
@@ -6,7 +6,18 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function getSum(a) {
+    count++;
+
+    return function getSum2(b) {
```

Comment:
Checklist #2: You don't need to name functions if you return them right away.
```suggestion
  return function getSum(a) {
    count++;

    return function(b) {
```